tests/tracking/fluent/test_fluent.py

```diff
@@ -423,6 +424,60 @@ def test_search_experiments(tmp_path):
     assert [e.name for e in experiments] == sorted(active_experiment_names, reverse=True)[:3]
 
 
+def test_search_registered_models(tmp_path):
+    sqlite_uri = "sqlite:///{}".format(tmp_path.joinpath("test.db"))
+    mlflow.set_tracking_uri(sqlite_uri)
+
+    num_all_models = SEARCH_REGISTERED_MODEL_MAX_RESULTS_DEFAULT + 1
+    num_azure_models = num_all_models // 4
+    num_boston_models = num_all_models - num_azure_models
+
+    azure_model_names = [f"AzureWeatherForecastModel_{i}" for i in range(num_azure_models)]
```

Comment:
Can we use a shorter name?

Reply:
Of course, just copied it from the docstring example. But I'll change it.